File: API/src/main/java/com/insane/apiwtb/model/Shop.java
package com.insane.apiwtb.model;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Builder
@Table(name = "TB_SHOP")
public class Shop {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id_shop")
    private int id;

    @Column(name = "nm_shop", nullable = false, length = 50)
    private String name;

    @Column(name = "ds_link", length = 100)
    private String link;

}

File: Mobile/Android/app/src/main/java/com/insane/whatsthebra/model/Shop.kt
package com.insane.whatsthebra.model

import com.insane.whatsthebra.database.dto.ShopDTO
import java.io.Serializable

class Shop (val id: Int = 0, var name: String, var link: String) : Serializable {

    fun toShopDTO(): ShopDTO {
        return ShopDTO (
            id = id,
            name = name,
            link = link
        )
    }

}